rest-api/tools/remove_service_account_keys.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+DATE=`date '+%Y-%m-%dT%H:%M:%SZ'`
+EXPIRE_DATE=`date -v"-3d"  "+%Y-%m-%d %H:%M:%S"`
+EXPIRE_DATE_EPOCH=`date -j -f"%Y-%m-%d %H:%M:%S" "$EXPIRE_DATE" +"%s"`
+PROJECTS="pmi-drc-api-test"
+
+for project in $PROJECTS
+do
+   echo "Project: $project"
+   for account in $(gcloud iam service-accounts list --project $project --format="value(email)")
+   do
+        # match any service account starting with awardee-
+        if [[ $account == awardee-* ]]
```

Comment:
We have a need to expire service account keys for SAs other than awardees. (Not sure whether it's all SAs, and whether the time after which we'd expire them would vary per SA... we need to figure this out.)

Meanwhile, I don't think we've actually settled on an approach for having awardees get SA keys that expire... they will be using them from systems that will need to work all the time. We need to follow up with Asmita and Neil about this.

So this is a great start, but let's hold off on merging it until we get more info.

Reply:
sounds good.